benchmark/plot-histogram.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I think you don't need this file if you use the sheband `#!/usr/bin/env gnuplot` on the gnuplot script.